# The form action data associated with an interactive card that was clicked. Only populated for CARD_CLICKED events.
string actionMethodName
google_chat_ros/ActionParameter[] parameters
